### PR TITLE
chore(plugins): bump version

### PR DESCRIPTION
**Pull request by abzcoding**
# Description

Update plugins

### CODE REVIEW

**Review by kylo252 — DISMISSED** [review]
LGTM! 🚀

File: lua/lvim/core/telescope.lua [hunk 1]
```diff
@@ -25,7 +25,16 @@ function M.config()
       layout_config = {
         width = 0.75,
         preview_cutoff = 120,
-        horizontal = { mirror = false },
+        prompt_position = "bottom",
```
Comment (kylo252) [inline lua/lvim/core/telescope.lua:28]: What was the reason behind this? It breaks other layouts, e.g. `<leader>Lf`

```suggestion
```

Comment (abzcoding) [reply lua/lvim/core/telescope.lua:28]: shit this was not supposed to be in here 😅

File: lua/lvim/core/telescope/custom-finders.lua [hunk 2]
```diff
@@ -14,6 +14,7 @@ local utils = require "lvim.utils"
 function M.find_lunarvim_files(opts)
   opts = opts or {}
```
Comment (kylo252) [inline lua/lvim/core/telescope/custom-finders.lua:15]: The problem is that we're merging over the theme.

Comment (abzcoding) [reply lua/lvim/core/telescope/custom-finders.lua:15]: ah i see, i wanted to make it like this
Screen Shot 2022-02-27 at 3 39 52 PM
Screen Shot 2022-02-27 at 3 40 04 PM

i'll revert the changes and do a separate one for custom styles

Comment (kylo252) [reply lua/lvim/core/telescope/custom-finders.lua:15]: Yeah, creating a custom style/theme should do it. It's tricky when modifying the global default.

**Review by kylo252 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**